core/errors.ml

```diff
@@ -7,14 +7,30 @@ type synerrspec = {filename : string; linespec : string;
                    message : string; linetext : string;
                    marker : string}
 
-exception UndefinedVariable of string
+type sugar_error_stage =
+  | DesugarFormlets
+  | DesugarRegexes
+  | CheckQuasiquotes
+  | DesugarLAttributes
+  | DesugarPages
+  | CheckXML
+
+let string_of_stage = function
```

Comment:
Alignment. In Emacs: `M-x align-regexp RET -> RET`

Reply:
Vim user, but I think using something like https://www.vim.org/scripts/script.php?script_id=294 is likely overdue :)

Reply:
> Vim user

Shame on you!